src/ontology/mondo-edit.obo

```diff
@@ -283660,11 +283660,11 @@ property_value: seeAlso "https://rarediseases.info.nih.gov/diseases/9965/hyperco
 [Term]
 id: MONDO:0012466
 name: Parkinson disease 13, autosomal dominant, susceptibility to
-def: "Any young-onset Parkinson disease in which the cause of the disease is a mutation in the HTRA2 gene." [MONDO:patterns/disease_series_by_gene]
+def: "An inherited susceptibility or predisposition to developing young-onset Parkinson disease, in which the cause of the disease is a mutation in the HTRA2 gene.." [MONDO:patterns/inherited_susceptibility]
 subset: predisposition
 synonym: "HTRA2 young-onset Parkinson disease" EXACT [MONDO:design_pattern, MONDO:patterns/disease_series_by_gene]
-synonym: "PARK13" RELATED ABBREVIATION [MONDO:Lexical]
-synonym: "Parkinson disease 13" EXACT []
+synonym: "PARK13" EXACT ABBREVIATION [MONDO:Lexical, OMIM:610297]
+synonym: "Parkinson disease 13" EXACT [OMIM:610297]
 synonym: "Parkinson disease 13, autosomal dominant, susceptibility to" EXACT [MONDO:Lexical, OMIM:610297]
 synonym: "susceptibility to autosomal dominant Parkinson disease 13" RELATED []
```

Comment:
this one should be exact based on the other synonym

Reply:
I removed "susceptibility to autosomal dominant Parkinson disease 13" RELATED []"
Because 
1- it doesn't have a source
2- it is confusing: the the Parkinson disease 13 autosomal dominant? or is the susceptibility autosomal dominant?